specification/2.0/Specification.adoc

```diff
@@ -1525,31 +1852,26 @@ The following examples shows a material that is defined using `pbrMetallicRoughn
 }
 ----
 
-[NOTE]
-.Implementation Note
-====
-If an implementation is resource-bound and cannot support all the maps defined it should support these additional maps in the following priority order.  Resource-bound implementations should drop maps from the bottom to the top.
+If a client implementation is resource-bound and cannot support all the textures defined it **SHOULD** support these additional textures in the following priority order.  Resource-bound implementations **SHOULD** drop textures from the bottom to the top.
 
 [options="header",cols="20%,80%"]
 |====
-| Map       | Rendering impact when map is not supported
+| Texture   | Rendering impact when feature is not supported
 | Normal    | Geometry will appear less detailed than authored.
-| Occlusion | Model will appear brighter in areas that should be darker.
+| Occlusion | Model will appear brighter in areas that are intended to be darker.
 | Emissive  | Model with lights will not be lit. For example, the headlights of a car model will be off instead of on.
 |====
 
-====
-
 
 [[alpha-coverage]]
 === Alpha Coverage
 
-The `alphaMode` property defines how the alpha value of the main factor and texture should be interpreted. The alpha value is defined in the `baseColor` for metallic-roughness material model.
+The `alphaMode` property defines how the alpha value is interpreted. The alpha value is taken from the fourth component of the _base color_ for metallic-roughness material model.
 
 `alphaMode` can be one of the following values:
 
 * `OPAQUE` - The rendered output is fully opaque and any alpha value is ignored.
-* `MASK` - The rendered output is either fully opaque or fully transparent depending on the alpha value and the specified alpha cutoff value. This mode is used to simulate geometry such as tree leaves or wire fences.
+* `MASK` - The rendered output is either fully opaque or fully transparent depending on the alpha value and the specified _alpha cutoff_ value. This mode is used to simulate geometry such as tree leaves or wire fences.
 * `BLEND` - The rendered output is combined with the background using the normal painting operation (i.e. the Porter and Duff over operator). This mode is used to simulate geometry such as guaze cloth or animal fur.
 
```

Comment:
I wonder whether the Porter and Duff Over-Operator has to be described/specificed. (This is similar to the point about the treatment of the MipMap lookups, in that it is usually beyond the control of the client implementation). However, here, it is not imposing any requirements like MUST/SHOULD, but just said "to be this way"...